src/pretix/base/payment.py

```diff
@@ -174,7 +174,7 @@ def settings_form_fields(self) -> dict:
 
         We suggest that you return an ``OrderedDict`` object instead of a dictionary
         and make use of the default implementation. Your implementation could look
-        like this::
```

Comment:
The "::" is intentional, as it signals in RST that a code block is about to begin. You can see the output here: https://docs.pretix.eu/en/latest/development/api/payment.html#pretix.base.payment.pretix.base.payment.BasePaymentProvider.BasePaymentProvider.settings_form_fields

Reply:
Thanks for explaining. I removed that change.